mlrun/api/main.py

```diff
@@ -56,6 +57,21 @@
 app.include_router(api_router, prefix=API_PREFIX, include_in_schema=False)
 
 
+def start_debugging() -> None:
+    if "enabled" not in os.environ.get("MLRUN_API_DEBUG_MODE", "").lower():
+        return
+
+    # pydevd_pycharm is installed only on development image. do not move this import to top
+    import pydevd_pycharm
+
+    pydevd_pycharm.settrace(
+        os.environ.get("MLRUN_API_DEBUG_HOST", "localhost"),
+        port=int(os.environ.get("MLRUN_API_DEBUG_PORT", 40000)),
```

Comment:
Same thing regarding @Tankilevitch's comment above. These should probably be in the config in a `debug` section